src/main/java/org/futureCollars/lesson6/task5/ReverseFileReader.java

```diff
@@ -0,0 +1,32 @@
+package org.futureCollars.lesson6.task5;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Stack;
+
+public class ReverseFileReader {
+
+    public static void main(String[] args) {
+        ReverseFileReader reader = new ReverseFileReader();
+        reader.readAndDislplayReversed("C:\\Tomek\\JAVA\\Java35_FilesExceptions\\src\\main\\resources\\data.txt");
```

Comment:
jak metoda ma w nazwie and to znaczy że robi za dużo rzeczy. Single responibility principle z zasad SOLID. może podziel to co jest w tej metodzie na dwa mniejsze metody. 1. pobierz treść treści z pliku 2. odwróć kolejność dzieki temu będziesz mógł stworzyć metodę np. readReversedFileContent w której wywołasz metodę z 1. i 2. jedną po drugiej. I zobacz jak ładnie podzielisz sobie kod na mniejsze reużuwalne elementy